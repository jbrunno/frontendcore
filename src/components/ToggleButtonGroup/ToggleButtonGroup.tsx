import React from 'react';
import {
  ToggleButtonGroup as MuiToggleButtonGroup,
  ToggleButtonGroupProps as MuiToggleButtonGroupProps,
} from '@mui/material';

export function ToggleButtonGroup(props: MuiToggleButtonGroupProps) {
  return <MuiToggleButtonGroup {...props} />;
}

export default ToggleButtonGroup;
