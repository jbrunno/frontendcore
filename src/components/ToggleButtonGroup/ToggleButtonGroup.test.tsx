import React from 'react';
import { render } from '@test-utils/component-testing';
import { ToggleButtonGroup } from './ToggleButtonGroup';

test('renders the ToggleButtonGroup', () => {
  const { container } = render(<ToggleButtonGroup />);
  expect(container).toMatchSnapshot();
});
