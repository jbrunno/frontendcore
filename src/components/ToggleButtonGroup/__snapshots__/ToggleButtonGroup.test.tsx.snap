// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the ToggleButtonGroup 1`] = `
.emotion-0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  border-radius: 4px;
}

.emotion-0 .MuiToggleButtonGroup-grouped:not(:first-of-type) {
  margin-left: -1px;
  border-left: 1px solid transparent;
  border-top-left-radius: 0;
  border-bottom-left-radius: 0;
}

.emotion-0 .MuiToggleButtonGroup-grouped:not(:last-of-type) {
  border-top-right-radius: 0;
  border-bottom-right-radius: 0;
}

.emotion-0 .MuiToggleButtonGroup-grouped.Mui-selected+.MuiToggleButtonGroup-grouped.Mui-selected {
  border-left: 0;
  margin-left: 0;
}

<div>
  <div
    class="MuiToggleButtonGroup-root emotion-0"
    role="group"
  />
</div>
`;
